src/hexen/p_enemy.c

```diff
@@ -4946,6 +4946,7 @@ void A_KoraxChase(mobj_t * actor)
             P_Teleport(actor, spot->x, spot->y, spot->angle, true);
         }
 
+        CheckACSPresent(249);
```

Comment:
This might be a larger problem in Hexen (I haven't looked), but might we eliminate these magic numbers? It seems that replacing `249` with `KORAX_TELEPORT_TID` will work here.